<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adultos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ni√±os</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbDias">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>------Seleccione------</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lunes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Miercoles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jueves</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Viernes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sabados</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Domingos</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spnAdultos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBoys">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAceptar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRespuesta">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
